Diagnostic/Utils/XmlUtil.py

```diff
@@ -26,18 +26,15 @@ def getXmlValue(xml,path,property):
     if element is not None:
         return element.get(property)
 
-def addElement(xml,path,el,selector=[]):
+
+def addElement(xml,path,el,addOnlyOnce=False,selector=[]):
```

Comment:
Good point, though I deliberately did this after checking that there's currently no addElement() calls with selector. Moved addOnlyOnce=False to the end. Thanks.